site-src/guides/migrating-from-ingress.md

```diff
@@ -0,0 +1,261 @@
+# Migrating from Ingress
+
+The Gateway API is not backward compatible with the [Ingress API](https://kubernetes.io/docs/concepts/services-networking/ingress/). As a result, if you are an Ingress user and would like to migrate, you will need to convert your Ingress resources to Gateway API resources. 
```

Comment:
Although this is largely true, it feels like a fairly negative way to start this guide. Maybe focus on the idea that Gateway API can generally be considered a superset of Ingress and is capable of much more than the original API.

Reply:
LGTM, thanks @shaneutt!

Reply:
thanks! I included the suggestion with some edits.